project/Build.scala

```diff
@@ -13,14 +13,11 @@ import scoverage.ScoverageSbtPlugin
 
 object Scrooge extends Build {
   val branch = Process("git" :: "rev-parse" :: "--abbrev-ref" :: "HEAD" :: Nil).!!.trim
-  val suffix = if (branch == "master") "" else "-SNAPSHOT"
```

Comment:
I'm not tied to a specific solution.  We can do that, but we should document it clearly and we should do it in util, finagle, ostrich, scrooge, and twitter-server at pretty much the same time so that it doesn't end up piecemeal.

Our feeling was that it would be useful to be always be able to build off of the `master` branch, and that folks who had to change the util / finagle / scrooge / ostrich / twitter-server `develop` branch would probably want it to be able to compile if you had published local snapshot versions of the other projects, so we made it depend on the `-SNAPSHOT` version and included instructions in the CONTRIBUTING.md file (looks like we forgot to make a CONTRIBUTING.md for scrooge though).  Our concern is that we'll sometimes add a new feature in util and then immediately depend on it in a different library, which works without problems in a monorepo, but has problems in a multi-repo scenario.

Maybe the right solution is to make a dbuild for twitter online-service repositories?  I haven't looked super deeply into dbuild.